modules/shared_data/src/main/resources/application.conf

```diff
@@ -31,7 +31,7 @@ rewards {
     vote-based-weight = 75
     initial-epoch = 0
     dao-address = "DAG6kfTqFxLLPLopHqR43CeQrcvJ5k3eXgYSeELt"
-    reward-calculation-interval = 100
+    reward-calculation-interval = 60
```

Comment:
@joaoCarvalho1000 @juandavidkincaid reward distribution interval had been changed, so instead of distribute it once in 100 minutes it is distribute once in 60 minutes (to be aligned to round duration)